### DISCUSSION TITLE
Question: 关于CDN

### DISCUSSION THREAD

**Discussion by Mr-Ao-Dragon** [LiterMC/go-openbmclapi#44] [Q&A] [answered]
此软件的CDN命中率如何，是否会大量触发回源？

**Comment by zyxkad** [comment, marked as answer]
Openbmclapi的访问路径大都是哈希路径，内容不会变，所以只要你的CDN缓存足够大可以做到一条都不回源

另外需要注意的是，缓存请求时需要忽略验证用的query参数

**Reply by Mr-Ao-Dragon** [reply]
好的，那理论上可以部署在FaaS服务里？

**Reply by zyxkad** [reply]
目前不可以，因为 Openbmclapi 程序需要通过一个websocket连接向主控发 keep-alive 表示服务还没有断

如果主控之后将主动保活变成全被动保活那么才有可能在无服务器函数里部署

**Reply by Mr-Ao-Dragon** [reply]
FaaS服务是随时在线的，我研究下各大提供商的ws
